src/preview/card.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+export function Card(props) {
```

Comment:
Você criou esse componente ``Card``. Inicialmente ele está sendo usado apenas uma vez. Contudo, se ele for requisitado em outros componentes, da para dar mais "dinamismo" para ele. Por exemplo, criar um ``CardHeader`` e um ``CardBody`` que recebem ``props``ou ``children``tornando-os dinâmicos. Qualquer dúvida entra em contato comigo que te mostro um exemplo bem legal.